### DISCUSSION TITLE
How can I drop the last layer to get output as hidden states in terms of feature extraction?

### DISCUSSION THREAD

**Discussion by Benan-Akca** [NVIDIA-NeMo/NeMo#2072] [Q&A]
**Describe your question**
Hi, I'm working on the Megatron model how can I drop the last layer to get output as hidden states in terms of feature extraction?

**Comment by ericharper** [comment]
This question is a little bit vague. Would you mind expanding on what you're trying to do? For example, what NeMo model are you working with and what would you like to do with the hidden states?
